src/languages/en.js

```diff
@@ -604,7 +600,7 @@ export default {
             avatarUploadFailureMessage: 'An error occurred uploading the avatar, please try again.',
         },
         error: {
-            growlMessageInvalidPolicy: 'Invalid workspace! You can create a new workspace!',
+            growlMessageInvalidPolicy: 'Invalid workspace! A new workspace has been created.',
```

Comment:
Verifying the ES translation for this still, will take off `HOLD` once confirmed.